fdm-core/src/cultivation.ts

```
@@ -282,59 +321,102 @@ export async function getCultivationPlan(fdm: FdmType, b_id_farm: schema.farmsTy
-            if (!existingCultivation) {
-                existingCultivation = {
-                    b_lu_catalogue: curr.b_lu_catalogue,
-                    b_lu_name: curr.b_lu_name,
-                    fields: []
-                };
-                acc.push(existingCultivation);
-            }
-
-            let existingField = existingCultivation.fields.find(field => field.b_id === curr.b_id);
+        const cultivationPlan = cultivations.reduce(
+            (acc: cultivationPlanType[], curr: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)